particles/fcc/particle_link.py

```diff
@@ -0,0 +1,27 @@
+from heppy.particles.particle_link import Particle_Link as BaseLink
+from pod import POD
+from heppy.papas.data.identifier import Identifier
+
+from ROOT import TLorentzVector
+from ROOT import TVector3
+
+class Particle_Link(BaseLink, POD):
+    '''Interface for link between two particles eg if a particle is the child/parent
+    of another particle.
+    The link contains only the uniqueid of each particle
+    '''    
+
+    def __init__(self, fccobj, parenttype='g', childtype='r'):
+        super(Particle_Link, self).__init__(fccobj)
+        self.incoming = []
+        self.outgoing = []
+        tlv = TLorentzVector()
+        #papas id for sim particle
+        p4 = fccobj.sim().core().p4
+        tlv.SetXYZM(p4.px, p4.py, p4.pz, p4.mass)
+        self._parentid= Identifier.make_id(Identifier.PFOBJECTTYPE.PARTICLE, fccobj.sim().getObjectID().index, parenttype, tlv.E())
```

Comment:
call this _p1  
(and _p2) 
so that it can be used for directed or undirected links

Reply:
OK